### PR TITLE
Fix/incorrect redis cmds

### LINKED ISSUE #40: List metrics does not return any new metrics pushed
List metrics of a bucket does not reflect any new metrics pushed.

### PR DESCRIPTION
**Pull request by rahulreddy**
- fix: flatten array for generating commands
Fix #40 

- test: assert basic metrics call

### CODE REVIEW

**Comment by rahulreddy** [comment]
@ironman-machine ...

**Comment by rahulreddy** [comment]
@ironman-machine 
SCALITY_S3_BRANCH= temp/utapiTesting

**Comment by michael-zapata** [comment]
Could we have the test asserted in the CI as well as more than a single one?

**Comment by rahulreddy** [comment]
@nicolas2bert already has opened PRs for the tests in Integration last week, they need some reviews.
As for the tests here, I am writing a test suite of unit and functional tests.

**Comment by rahulreddy** [comment]
Added two additional tests.

File: package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
     "ft_test": "mocha --compilers js:babel-core/register --recursive tests/functional",
     "lint": "eslint $(git ls-files '*.js')",
     "lint_md": "mdlint $(git ls-files '*.md')",
-    "start": "node index.js"
+    "start": "node index.js",
+    "test": "mocha --compilers js:babel-core/register --recursive tests/unit"
```
Comment (michael-zapata) [inline package.json]: Can we have this run into the CI itself?

Comment (rahulreddy) [inline package.json]: Good catch, I have added npm test to circle.yml

Comment (rahulreddy) [inline package.json]: I just realized you commented on the rebase of other PR. I am moving this change over there.

**Comment by LaurenSpiegel** [comment]
👍

File: lib/backend/Memory.js [hunk 2]
```diff
@@ -144,11 +155,16 @@ export default class Memory {
     * Store value by score in a sorted set
     * @param {string} key - data key
     * @param {number} score - data score
-    * @param {number} value - data value
+    * @param {string} value - data value
```
Comment (michael-zapata) [inline lib/backend/Memory.js]: Value can also be a number here as well.

Comment (rahulreddy) [inline lib/backend/Memory.js]: Changed.

**Comment by rahulreddy** [comment]
@ironman-machine ...

**Pull request merged**